addons/survey/models/survey.py

```diff
@@ -0,0 +1,1066 @@
+    ''' Questions that will be asked in a survey.
+
+    Each question can have one of more suggested answers (eg. in case of
+    dropdown choices, multi-answer checkboxes, radio buttons...).'''
+    _name = 'survey.question'
+    _description = 'Survey Question'
+    _rec_name = 'question'
+    _order = 'sequence,id'
+
+    # Model fields #
+    # Question metadata
+    page_id = fields.Many2one('survey.page', string='Survey page',
+        ondelete='cascade', required=True, default=lambda self: self.env.context.get('page_id'))
+    survey_id = fields.Many2one('survey.survey', related='page_id.survey_id', string='Survey')
+    sequence = fields.Integer(default=10)
+
+    # Question
+    question = fields.Char(string='Question Name', required=True, translate=True)
+    description = fields.Html(help="Use this field to add \
+        additional explanations about your question", translate=True,
+        oldname='descriptive_text')
+
+    # Answer
+    question_type = fields.Selection([('free_text', 'Multiple Lines Text Box'),
+            ('textbox', 'Single Line Text Box'),
+            ('numerical_box', 'Numerical Value'),
+            ('datetime', 'Date and Time'),
+            ('simple_choice', 'Multiple choice: only one answer'),
+            ('multiple_choice', 'Multiple choice: multiple answers allowed'),
+            ('matrix', 'Matrix')], string='Type of Question', size=15, required=True, default='free_text')
+    matrix_subtype = fields.Selection([('simple', 'One choice per row'),
+        ('multiple', 'Multiple choices per row')], string='Matrix Type', default='simple')
+    labels_ids = fields.One2many('survey.label',
+        'question_id', string='Types of answers', oldname='answer_choice_ids', copy=True)
+    labels_ids_2 = fields.One2many('survey.label',
+        'question_id_2', string='Rows of the Matrix', copy=True)
+    # labels are used for proposed choices
+    # if question.type == simple choice | multiple choice
+    #                    -> only labels_ids is used
+    # if question.type == matrix
+    #                    -> labels_ids are the columns of the matrix
+    #                    -> labels_ids_2 are the rows of the matrix
+
+    # Display options
+    column_nb = fields.Selection([('12', '1'),
+                                   ('6', '2'),
+                                   ('4', '3'),
+                                   ('3', '4'),
+                                   ('2', '6')],
+        string='Number of columns', default='12')
+        # These options refer to col-xx-[12|6|4|3|2] classes in Bootstrap
+    display_mode = fields.Selection([('columns', 'Radio Buttons'),
+                                      ('dropdown', 'Selection Box')], default='columns')
+
+    # Comments
+    comments_allowed = fields.Boolean(string='Show Comments Field',
+        oldname="allow_comment")
+    comments_message = fields.Char(string='Comment Message', translate=True, default='If other, precise:')
```

Comment:
May be we should return translated term using _ (create a function to return value), so that this default string get translated